shopinvader/views/shopinvader_backend_view.xml

```diff
@@ -59,14 +59,24 @@
                         <page name="sale" string="Sale">
                             <group name="cart_conf" string="Cart configuration">
                                 <field name="clear_cart_options"/>
-                                <field name="anonymous_partner_id"/>
                                 <field name="pricelist_id" required="1"/>
                             </group>
                             <group name="sale_conf" string="Sale configuration">
                                 <field name="company_id"/>
                                 <field name="sequence_id"/>
                                 <field name="account_analytic_id"/>
                             </group>
+                            <group name="customer" string="Customer configuration">
```

Comment:
@simahawk If you add new "customer" group it would be nice to also put the configuration fields from `shopinvader_guest_mode` into this new group https://github.com/shopinvader/odoo-shopinvader/blob/12.0/shopinvader_guest_mode/views/shopinvader_backend.xml#L12